### ISSUE TITLE
accessibility: down arrow jumps over several messages instead of moving one by one 

### ISSUE THREAD

**Issue by vortex1024** [UnigramDev/Unigram#2791]
**Describe the bug**
in the message list, down arrow moves down several messages. It seems this doesn't happen if you are very high up in the list, with many items remaining. I wasn't able to determine the threshhold from where this starts happening
I tested only with screen readers on, as I can't use the pc otherwise.
**To Reproduce**
1. open narrator
2. open Unigram
3. open a chat, focus the message list
4. arrow up 5 times
5. arrow down once
6. focus will go to the last message in the chat
**Expected behavior**
focus should have moved to the 4th last message 

**Screenshots**

**Version Info**
 - Unigram: .UnigramPreview version 8.8.7438.0
 - Windows: 21H2 (OS Build 19044.

**Additional context**

**Comment by makisukurisu** [comment]
@vortex1024 This was fixed in newest version - 7447, try updating your app in Microsoft Store, note that since all updates are released gradually it may take a few days for you to get the update.

As a temporary solution you may use Shift+Down Arrow combination to solve this problem.

Thanks for your report.

P.S. May I ask you to confirm that the problem was solved after you get the latest update?

**Comment by FrayxRulez** [comment]
Fixed in https://github.com/UnigramDev/Unigram/commit/a012543b774ab4db2a07da2c4c4b086f4a764da1
